projects/swaypad/Forc.toml

```diff
@@ -5,5 +5,6 @@ license = "Apache-2.0"
 name = "swaypad"
 
 [dependencies]
-standards = { git = "https://github.com/FuelLabs/sway-standards", tag = "v0.7.0" }
-sway_libs = { git = "https://github.com/FuelLabs/sway-libs", tag = "v0.25.2" }
```

Comment:
We shouldn't be removing `sway_libs` entirely. We need to granular import each of the libraries from sway_libs from `forc.pub`.  Unfortunately it seems the libraries that were published were built against sway 0.69.0 which is ahead of the supported mainnet release of `0.68.9`. @bitzoic any chance we could get an earlier version of the libraries published that build against `0.68.9`?

Reply:
we also want to include every single `src` version here so they are accessible to use in sway playground.

Reply:
done in [92c6505](https://github.com/FuelLabs/sway-playground/pull/125/commits/92c6505f20cc4561c9dc398090ea9b0254825df3)